ml-agents/mlagents/trainers/trainer.py

```diff
@@ -17,6 +19,88 @@ class UnityTrainerException(UnityException):
     pass
 
 
+class TrainerMetrics(object):
```

Comment:
I'd suggest we move this class into a separate file and add some unit tests for its behavior.

Reply:
I think the class is fine where it is. It is a helper function that is tied to `Trainer`

Reply:
I would like it in a separate file as well...

Reply:
will be fixed